icaruscode/CRT/crtsimmodules_icarus.fcl

```diff
@@ -85,7 +85,7 @@ icarus_crtsim:
 {
   module_type: "icaruscode/CRT/CRTDetSim"
 
-  G4ModuleLabel: "largeant"
+  G4ModuleLabel: "genericcrt"
```

Comment:
Is this the intended behavior?

Reply:
yes, this is part of making refactored G4 the default in the upstream fcl files. 

Here the confusion may be that what is referred as "G4ModuleLabel" is not the G4 module in the refactored scheme.